openedx/core/djangoapps/ace_common/message.py

```diff
@@ -1,13 +1,54 @@
 """
 Base Message types to be used to construct ace messages.
 """
-from edx_ace.message import MessageType
+from django.utils.functional import Promise
+from edx_ace.message import MessageType, Message
+from edx_ace.serialization import MessageEncoder
+import json
 from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
 
 
+class EdxMessageEncoder(MessageEncoder):
```

Comment:
edx-ace is our own code.  Why don't we just add this support in the original repo? It already imports Django in plenty of places.  It would be a 4-line improvement there, rather than a fragile 40-line override here.

Reply:
PR created https://github.com/edx/edx-ace/pull/34